doc/ref_impl/cntt-ri/chapters/chapter02.md

```diff
@@ -5,15 +5,75 @@
 
 ## Table of Contents
 * [2.1 Introduction](#2.1)
-* [2.2 Heading](#2.2)
-* [2.3 Heading](#2.3)
+* [2.2 Reference Architecture Requirement](#2.2)
+* [2.3 Reference Implementation Requirement](#2.3)
+* [2.4 Typical Example for Reference Implementation](#2.3)
+
+<a name="2.1"></a>、
+
 
-<a name="2.1"></a>
 ## 2.1 Introduction
 
+
+**must**: Requirements that are marked as _must_ are considered mandatory and must exist in the reference architecture and reflected in any implementation targeting this reference architecture. The same applies to _must not_.
+
+**should**: Requirements that are marked as _should_ are expected to be fulfilled by the reference architecture but it is up to each service provider to accept an implementation targeting this reference architecture that is not reflecting on any of those requirements. The same applies to _should not_.
+> RFC2119
+
+**may**: Requirements that are marked as _may_ are considered optional. The same applies to _may not_.
```

Comment:
__Query__ - We are redefining these terms at a lot of places. We should keep it at one place (RM Glossary?) with a reference to RFC2119 there only unless we deviate from the terminology. Then we explicitly mention it out. Opinion?

Reply:
seems covered by the commit below and based on @deebhatia approval, I will resolve